packages/cli/cli/src/commands/generate/generateAPIWorkspace.ts

```diff
@@ -59,6 +59,9 @@ export async function generateWorkspace({
             context
         });
     } else {
+        if (!token) {
+            return context.failAndThrow("Must provide token if 'useLocalDocker' is false");
```

Comment:
`useLocalDocker` is not a flag the user knows about -- they only know about `--local`

Reply:
how about `Please run fern login.`